core/base/implicitTriangulation/ImplicitTriangulation.h

```diff
@@ -267,6 +267,8 @@ namespace ttk {
 #endif // TTK_ENABLE_MPI
 
   protected:
+    std::array<ttk::SimplexId, 64 * 14> neighborIndexLUT;
```

Comment:
also, could you please rename this variable into `neighborIndexLUT_` (member variables in the base code have a final _)